backend/fleet_management/admin.py

```diff
@@ -18,6 +18,9 @@ def dehydrate_passengers(self, drive):
             for passanger in drive.passengers.all()
         )
 
+    def dehydrate_driver__country(self, drive):
+        return drive.driver.country.name
+
```

Comment:
I am wondering if it should be name or a symbol. We can change it later if needed anyways ;)

Reply:
That's right :) I assumed that since it is a report and it has not been clearly indicated that it is to be a country code, it is likely that it should be the full name of the country.